documentation/asciidoc/topics/proc_configuring_sites_manually.adoc

```diff
@@ -0,0 +1,106 @@
+[id='configuring_sites-{context}']
+= Specifying {brandname} Cluster Hostnames for Cross-Site Replication
+Specify hostnames and ports for {brandname} clusters so they can establish connections and form cross-site views.
+
+.Prerequisites
+
+* Ensure that you have the correct hostnames and port numbers for each {brandname} cluster that you plan to configure as a backup location.
+
+.Procedure
+
+. Create an **Infinispan** CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Provide the name, URL, and port for each {brandname} cluster that acts as a backup location with `spec.service.sites.locations`.
++
+The following are example **Infinispan** CR definitions for **LON** and **NYC**:
++
+* **LON**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 3
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: LON
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: LON
+          host: infinispan-lon.myhost.com
+          port: 7900
+        - name: NYC
+          host: infinispan-nyc.myhost.com
+          port: 7900
+----
++
+* **NYC**
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: {example_crd_name}
+spec:
+  replicas: 2
+  service:
+    type: DataGrid
+    sites:
+      local:
+        name: NYC
+        expose:
+          type: LoadBalancer
+      locations:
+        - name: NYC
+          url: infinispan-nyc.myhost.com
+          port: 7900
+        - name: LON
+          url: infinispan-lon.myhost.com
+          port: 7900
+----
++
+. Adjust logging levels for cross-site replication as follows:
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+spec:
+  ...
+  logging:
+    categories:
+      org.jgroups.protocols.TCP: error
+      org.jgroups.protocols.relay.RELAY2: fatal
+----
++
+The preceding configuration decreases logging for JGroups TCP and RELAY2
+protocols to reduce excessive messages about cluster backup operations, which
+can result in a large number of log files that use container storage.
++
+. Configure nodes with any other {datagridservice} resources.
+. Apply the **Infinispan** CRs.
+. Check node logs to verify that {brandname} clusters form a cross-site view, for example:
++
+[source,options="nowrap",subs=attributes+]
+----
+$ {oc_logs} {example_crd_name}-0 | grep x-site
+
+INFO  [org.infinispan.XSITE] (jgroups-5,{example_crd_name}-0-<id>) ISPN000439: Received new x-site view: [NYC]
+INFO  [org.infinispan.XSITE] (jgroups-7,{example_crd_name}-0-<id>) ISPN000439: Received new x-site view: [NYC, LON]
+----
+
+.Next steps
+
+If your clusters have formed a cross-site view, you can start adding backup
```

Comment:
For static xsite no specific cross-site view validations are performed

Reply:
@dmvolod Thanks for the review comments. I swear I thought I had all the instances of `url` changed to `host`. Maybe I dreamed it. 

Don't users still get "new x-site view" messages in logs even if it's a static configuration?

Reply:
@oraNod I don't see message like you provided inside code, just `x-site configured`
https://github.com/infinispan/infinispan-operator/blob/3948a1c1c354b679ffcd66b6e12ec6bc0ae7d501/pkg/controller/infinispan/xsite.go#L66

Reply:
thanks @dmvolod 

@rigazilla and @ryanemerson shouldn't the message logged for static xsite be the same as for automatic xsite? that is to say the message should be "Received new x-site view:"

Reply:
the server logs are independent of the operator. it logs the x-site view (ISPN000439 message)

Reply:
thanks @pruivo this is what I mean. you can still get the xsite view from the server to verify the configuration. 

@dmvolod is there something else I'm missing or that we need to change?

Reply:
Looks good, thanks @oraNod